cognite/extractorutils/util.py

```diff
@@ -197,24 +197,27 @@ def add_extraction_pipeline(
     added_message: str = "",
 ) -> Callable[[Callable[..., _T1]], Callable[..., _T1]]:
     """
-    This is to be used as a decorator for extractor functions to add extraction pipeline information
+    This is to be used as a decorator for extractor functions to add extraction pipeline information.
 
     Args:
-        extraction_pipeline_ext_id:
-        cognite_client:
-        heartbeat_waiting_time:
-        added_message:
+        extraction_pipeline_ext_id: Id of the extraction pipeline
+        cognite_client: Assets to create
+        heartbeat_waiting_time: Interval of the execution of the Thread
+        added_message: Message that will be displayed at the end of the run
 
     Usage:
         If you have a function named "extract_data(*args, **kwargs)" and want to connect it to an extraction
         pipeline, you can use this decorator function as:
-        @add_extraction_pipeline(
-            extraction_pipeline_ext_id=<INSERT EXTERNAL ID>,
-            cognite_client=<INSERT COGNITE CLIENT OBJECT>,
-            logger=<INSERT LOGGER>,
-        )
-        def extract_data(*args, **kwargs):
-            <INSERT FUNCTION BODY>
+
+        .. code-block:: python
+
+            @add_extraction_pipeline(
+                extraction_pipeline_ext_id=<INSERT EXTERNAL ID>,
+                cognite_client=<INSERT COGNITE CLIENT OBJECT>,
+                logger=<INSERT LOGGER>,
```

Comment:
I see this was from the earlier text as well, but it does not seem to me that we expect a logger argument here?

``` suggestion
```

Reply:
Good catch, no there isn't, right now there is only the `added message` parameter, wich is a str

Reply:
done